.github/workflows/build_for_riscv.yml

```
@@ -58,33 +58,169 @@
+        
+        # Make binaries executable
+        sudo chmod +x /tmp/wasmedge-riscv64/bin/*
+        sudo chmod +x /tmp/wasmedge-riscv64/run_tests.sh
+        
+        # Set up library path for RISC-V64 libraries
+        export QEMU_LD_PREFIX=/usr/riscv64-linux-gnu
+        
+        # Run the test script through QEMU
+        echo "=== Running WasmEdge tests on RISC-V64 via QEMU ==="
+        qemu-riscv64-static -L /usr/riscv64-linux-gnu /tmp/wasmedge-riscv64/run_tests.sh
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)